### DISCUSSION TITLE
The ability to add (custom) quotes

### DISCUSSION THREAD

**Discussion by Farzat07** [monkeytypegame/monkeytype#3879] [Ideas]
I tried the normal "custom" option, but that seemed to imitate "time" and "words" options more. What I'm looking for is the ability to add a bunch custom "quotes" which circulates, and some of which could possibly be added to the official quotes list.

**Comment by ajeetgupta80** [comment]
now its added you can explore the customs tab on home screen

**Comment by unrealapex** [comment]
You could just supply them in the `custom text` text area and put a new line between the quotes you want to type.

**Reply by Farzat07** [reply]
This doesn't really work as:

- It isn't as easy to add and save the quotes.
- The quotes aren't shown in random order.

**Reply by unrealapex** [reply]
Monkeytype does have a quote saving feature:

![quote saving](https://user-images.githubusercontent.com/79121360/235957038-6d539ebe-cf1f-4a10-9f91-f13003e2ca1b.png)
If you want the quotes to be in random order, set `Random` quote to be by **`Time`**.

**Comment by flonle** [comment]
I would love to review my collection of quotes while on Monkeytype, but the absence of this feature disallows this currently.

The custom text route is not satisfactory. They aren't separate quotes, just blobs to text separated by new newlines. A long list of quotes will just keep going endlessly instead of showing your score after each one. They are also not randomized, making it completely unusable for anything more than a few quotes.

A private quote feature would be an amazing addition. Right now, I couldn't even submit new quotes if I wanted to, since the feature is disabled because of a large submission queue. Private quotes should be normal quotes, but exempt from the submission pipeline and private to the account.
